pricing-page-exercise/salih/index.html

```diff
@@ -1,13 +1,121 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <link rel="stylesheet" href="./styles.css">
-  <title>Pricing page Exercise</title>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>getbootstrap Pricing Page</title>
+    <link rel="stylesheet" href="styles.css">
 </head>
+
 <body>
-  
+    <nav>
+        <div class="nav-container">
+            <div class="company-name">
+                <span>Company Name</span>
+            </div>
+            <div class="menu-container">
+                <div class="links">
+                    <ul class="menu" >
+                        <a href="#"><li class="menu-item">Features</li></a>
+                        <a href="#"><li class="menu-item">Enterprice</li></a>
+                        <a href="#"><li class="menu-item">Support</li></a>
+                        <a href="#"><li class="menu-item">Pricing</li></a>
+                    </ul>
+                </div>
+                <div class="sign-up-button">
+                    <a class="blue" href="#">Sign Up</a>
```

Comment:
I'd recommend to have this <a> element inside the unordered list above. You can keep it just as another item of the list. This would be more semantically correct and should not really affect styling as long as you add different classes to this element and other link elements in the list.

Reply:
Done. I also changed `.links` css code to adapt this changement.